src/main/config/config.c

```diff
@@ -111,6 +111,9 @@ PG_REGISTER_WITH_RESET_TEMPLATE(pilotConfig_t, pilotConfig, PG_PILOT_CONFIG, 2);
 PG_RESET_TEMPLATE(pilotConfig_t, pilotConfig,
     .craftName = { 0 },
     .pilotName = { 0 },
+    .message[0] = { 0 },
```

Comment:
IIRC this should do:
```suggestion
    .message = { 0 },
```